webrtc-stats.html

```diff
@@ -1142,6 +1143,21 @@ <h2>
                   <code><a>packetsDiscarded</a></code>.
                 </p>
               </dd>
+              <dt>
+                <dfn><code>perDscpPacketsReceived</code></dfn> of type <span class="idlMemberType"><a>record&lt;USVString, unsigned long></a></span>
+              </dt>
+              <dd>
+                <p>
+                  Total number of packets received for this SSRC, per
+                  Diferentiated Services code point (DSCP)
+                  [[RFC2474]]. DSCPs are
+                  identified as decimal integers in string form. Note
+                  that due to network remapping and bleaching, these
+                  numbers are not expected to match the numbers seen
+                  on sending. Not all OSes make this information
+                  available.
```

Comment:
When OSes don't provide this information, should the member be absent or present with an empty map?